src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/PowerLauncherPluginSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.PowerToys.Settings.UI.Library
+{
+    public class PowerLauncherPluginSettings
```

Comment:
Have you missed to add a property for the plugin icon? I think we need it later for the plugin list in the settings.

Reply:
Good catch! 
Regarding `Name` property and some others, we can make them readonly.